code/modules/hydroponics/grown/apple.dm

```diff
@@ -58,3 +58,35 @@
 	desc = "Emblazoned upon the apple is the word 'Kallisti'."
 	icon_state = "goldapple"
 	filling_color = "#FFD700"
+	
+	// Brass Apple
+/obj/item/seeds/apple/gold/brass //it looks sorta gold-y so its a subtype okay?
+	name = "pack of brass apple seeds"
+	desc = "These seeds grow into brass apple trees. Store away from machinery."
+	species = "brassapple"
+	plantname = "Brass Apple Tree"
+	product = /obj/item/reagent_containers/food/snacks/grown/apple/gold/Brass
+	mutatelist = list(/obj/item/seeds/apple/gold/brass/charged)
+	reagents_add = list("brass" = 0.2, "vitamin" = 0.04, "nutriment" = 0.1)
+
+/obj/item/reagent_containers/food/snacks/grown/apple/gold/brass
+	seed = /obj/item/seeds/apple/gold/brass
+	name = "brass apple"
+	desc = "You could swear you hear a faint ticking."
+	filling_color = "#B5A642"
+
+	// Charged Brass Apple
+/obj/item/seeds/apple/gold/brass/charged
+	name = "pack of charged brass apple seeds"
+	desc = "These seeds grow into charged brass apple trees. Handle with extreme care."
+	species = "chargedbrassapple"
+	plantname = "Charged Brass Apple Tree"
+	product = /obj/item/reagent_containers/food/snacks/grown/apple/gold/Brass
```

Comment:
is it charged or not jesus christ can you please do this through dreammaker why do you people think this is ok

Reply:
tell me exactly how to get byond to run on a google chromebook and i will.